boa_engine/src/syntax/ast/expression/spread.rs

```diff
@@ -24,29 +24,32 @@ use super::Expression;
 #[cfg_attr(feature = "deser", serde(transparent))]
 #[derive(Clone, Debug, PartialEq)]
 pub struct Spread {
-    val: Box<Expression>,
+    expression: Box<Expression>,
 }
 
 impl Spread {
+    /// Gets the expression to be expanded by the spread operator.
     #[inline]
     pub fn val(&self) -> &Expression {
-        &self.val
+        &self.expression
     }
 
     /// Creates a `Spread` AST Expression.
     #[inline]
     pub fn new(val: Expression) -> Self {
```

Comment:
I guess here too, the name of the variable should be `expression` or something like that.